package.json

```diff
@@ -40,9 +40,9 @@
     "test.headless.ci": "nx component-test-ci headless",
     "test.utils": "nx test utils"
   },
-  "packageManager": "pnpm@9.1.1",
+  "packageManager": "pnpm@8.15.8",
```

Comment:
Any reason why we're using an older pnpm version here?

Reply:
Sorry forgot to give context.

Using pnpm 8.15.8 because of https://github.com/qwikifiers/qwik-ui/issues/744

Updated first message.